setup.py

```diff
@@ -23,6 +23,7 @@
         "onnxruntime>=1.10.0",
         "paramiko==2.11.0",
         "torch>=1.12.1",
+        "torch_geometric>=2.3.0",
```

Comment:
`torch_geometric` is a requirement for GNN models, not for mlagility. Therefore, this should be at `models/requirements.txt`.